pages/storage_and_backup/object_storage/cold_archive_overview/guide.de-de.md

```diff
@@ -39,13 +45,15 @@ The service allows you to focus on building and deploying cloud applications whi
 
 The service is fully managed by OVHcloud and accessible through S3 API.
 
-**3 steps operation**
+**5 steps operation**
```

Comment:
'5 steps operation' is a bit misleading in my opinion.
It is not exactly one operation comprised of 5 steps but rather there are 3 possible operations : archiving, restoring, and deleting.

Reply:
Well, ok you're right. We should maybe use a "3-operation process: Store - Archive - Restore" instead, and explain there are other actions (move, delete..) also available. What do you think @lason-ovh?